src/main/java/uk/gov/justice/hmpps/prison/api/model/PrisonerSearchDetails.kt

```diff
@@ -68,6 +71,9 @@ data class PrisonerSearchDetails(
   @Schema(description = "Prisoner Identifiers")
   val identifiers: List<OffenderIdentifier>? = null,
 
+  @Schema(description = "Prisoner Identifiers including those from aliases")
+  val allIdentifiers: List<OffenderIdentifier>? = null,
+
```

Comment:
Add `allIdentifiers` to the response object which will eventually replace `identifiers`